src/GUI/ErikEJ.EntityFrameworkCore.5.Edmx/ErikEJ.EntityFrameworkCore.5.Edmx.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <RootNamespace>ErikEJ.EntityFrameworkCore.Edmx</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\RevEng.Core.Abstractions\RevEng.Core.Abstractions.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Scaffolding\" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\ErikEJ.EntityFrameworkCore.Edmx\Scaffolding\EdmxDatabaseModelFactory.cs">
+      <Link>Scaffolding\EdmxDatabaseModelFactory.cs</Link>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="linq2edmx" Version="0.0.1-pre" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="5.0.1" />
```

Comment:
5.0.0 ?

Reply:
Sorry I was in the process of writing a big comment.
I removed the Microsoft.EntityFrameworkCore.Relational reference

Reply:
And now fixed it to 5.0.0.
Is that what you'd expect?
Could you clarify "unneeded inline code comments"?